source/includes/fundamentals/code-snippets/builders/AggBuilders.java

```diff
@@ -88,6 +89,15 @@ private void aggregationExample() {
         // end sampleAggregation
     }
 
+    private void setWindowFieldsStage() {
+        // begin setWindowFields
+        Window pastMonth = Windows.timeRange(-1, Windows.Bound.CURRENT, Unit.MONTH);
+        setWindowFields("$localityId", Sorts.ascending("measurementDateTime"),
+                WindowedComputations.sum("monthlyRainfall", "$rainfall", pastMonth),
+                WindowedComputations.avg("monthlyAvgTemp", "$temp", pastMonth));
```

Comment:
```suggestion
                WindowedComputations.avg("monthlyAvgTemp", "$temperature", pastMonth));
```
Nit: For consistency since you refer the variable as `temperature` in the description